From 1ead10d2b214ec5545ee2cdb25d5b0cdb3b4f735 Mon Sep 17 00:00:00 2001
From: Josh Poimboeuf <jpoimboe@redhat.com>
Date: Wed, 17 Aug 2022 12:10:39 -0700
Subject: [PATCH 1/3] kpatch-cc: Add more file ignores

These files aren't in the kernel proper, and can be ignored.

Signed-off-by: Josh Poimboeuf <jpoimboe@redhat.com>
---
 kpatch-build/kpatch-cc | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/kpatch-build/kpatch-cc b/kpatch-build/kpatch-cc
index 6f0063e..d2bccdd 100755
--- a/kpatch-build/kpatch-cc
+++ b/kpatch-build/kpatch-cc
@@ -31,22 +31,22 @@ if [[ "$TOOLCHAINCMD" =~ ^(.*-)?gcc$ || "$TOOLCHAINCMD" =~ ^(.*-)?clang$ ]] ; th
 				vmlinux.o|\
 				.tmp_kallsyms1.o|\
 				.tmp_kallsyms2.o|\
-				init/version.o|\
-				arch/x86/boot/version.o|\
-				arch/x86/boot/compressed/eboot.o|\
-				arch/x86/boot/header.o|\
-				arch/x86/boot/compressed/efi_stub_64.o|\
-				arch/x86/boot/compressed/piggy.o|\
-				kernel/system_certificates.o|\
-				arch/x86/vdso/*|\
+				arch/x86/boot/*|\
 				arch/x86/entry/vdso/*|\
-				drivers/firmware/efi/libstub/*|\
+				arch/x86/purgatory/*|\
+				arch/x86/realmode/*|\
+				arch/x86/tools/*|\
+				arch/x86/vdso/*|\
 				arch/powerpc/kernel/prom_init.o|\
 				arch/powerpc/kernel/vdso64/*|\
 				arch/s390/boot/*|\
 				arch/s390/purgatory/*|\
 				arch/s390/kernel/vdso64/*|\
+				drivers/firmware/efi/libstub/*|\
+				init/version.o|\
+				kernel/system_certificates.o|\
 				lib/*|\
+				tools/*|\
 				.*.o|\
 				*/.lib_exports.o)
 					break
--
2.35.5

